src/dmqproto/client/UsageExamples.d

```diff
@@ -97,7 +97,10 @@ version ( UnitTest )
                         info.error_while_connecting.node_addr.port);
                     break;
 
-                default:
+                case none:
```

Comment:
You can use [`SmartUnion.handleInvalidCases`](https://github.com/sociomantic-tsunami/ocean/blob/v3.x.x/src/ocean/core/SmartUnion.d#L334) here.